src/Http/Requests/MutateRequest.php

```diff
@@ -69,18 +70,15 @@ protected function relationRules(Resource $resource, string $prefix = '', $loade
         ) {
             $prefixRelation = $prefix.'.'.$relation->relation;
 
-            if ($relation instanceof BelongsToMany || $relation instanceof HasMany) {
+            if ($relation instanceof BelongsToMany || $relation instanceof HasMany || $relation instanceof HasManyThrough) {
                 $prefixRelation .= '.*';
             }
```

Comment:
Need to refactor this in order to define something in the relation that indicate that its waiting for an array instead of an object